app/views/contributions/_contribution.html.erb

```diff
@@ -28,8 +28,6 @@
         </small>
       </span>
     </h4>
-    <p class="js-emoji">
-      <%= truncate contribution.body, :length => 240 %>
-    </p>
+    <%= simple_format Markdown.new(truncate contribution.body, :length => 240).to_html.gsub(/(\\n)/, '<br>'), class: "js-emoji contibution-body", wrapper_tag: "div" %>
```

Comment:
Can we make a helper method for this as it's used three times, will also make it easier to test

Reply:
Helper method created. My only question is how would I test this? I am very new to rails and rspec and my ability to picture how to test things is coming along a lot slower than how to implement them in the first place. :(

Reply:
There's a few example over here: https://github.com/24pullrequests/24pullrequests/blob/main/spec/helpers/application_helper_spec.rb

Reply:
I added a test for replacing the \n and \r characters. Should I be testing if the markdown gets converted too? Or is that not more a part of commonmarker so it doesn't need to be tested in this app?